### ISSUE TITLE
New Kahoot Two factor option

### ISSUE THREAD

**Issue by msemple1111** [unixpickle/kahoot-hack#73]
kahoot have added a new option for educators to play with a two factor code. This code changes every  7 seconds. It is relatively easy to fix for kahoot-play and kahoot-profane, just add a command line feature to ask for the order. 

For kahoot-flood and kahoot-rand it looks more difficult. One option could be to keep asking for two factor codes but this is quite laborious for the player.

**Comment by tehrealblackout** [comment]
How can I run this on My iPhone or can you only do it on a pc

**Comment by msemple1111** [comment]
@tehrealblackout I don't think this is related to this issue. As of now, no you cannot run a kahoot hack on the iPhone.

**Comment by msemple1111** [comment]
https://camo.githubusercontent.com/1082d8cd8248c5b13401d0e3213e91d1b6be99b1/687474703a2f2f692e696d6775722e636f6d2f757076373548382e706e67

**Comment by GRIMMboi** [comment]
this would be a great thing to fix
